### PR TITLE
Prep. for PHP 5.3 / 2.0 of the project

### PR DESCRIPTION
**Pull request by SpacePossum**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,8 +20,7 @@ See Traits and asserts listing for more details.
 
 ### Requirements
 
-PHP 5.4 (PHP7 supported). Optional HHVM support >= 3.9.
-PHPUnit >= 3.5.0
+PHP 5.4 (PHP 5.3 for the constraints, PHP 7 supported). Optional HHVM support >= 3.9. PHPUnit >= 3.5.0.
```
Comment (keradus) [inline README.md]: 5.3 -> 5.3.6
there is a big difference between them

Comment (SpacePossum) [inline README.md]: good point!

**Comment by keradus** [comment]
sorry for asking you to rewrite the package :(

**Comment by SpacePossum** [comment]
>sorry for asking you to rewrite the package :(

You didn't, no worries man! I'm doing a bit of housekeeping as I go :)

I have some other ideas as well, and besides; one the asserts in this package (`isDirectory`) has been added to PHPUnit itself so I've to see what I do with that.

**Comment by SpacePossum** [comment]
Thank you @keradus :)

**Comment by keradus** [comment]
A lot of great work here! Thanks !

**Pull request merged**